src/services/cssCompletion.ts

```diff
@@ -558,14 +558,11 @@ export class CSSCompletion {
 			});
 		}
 		for (const p of languageFacts.colorFunctions) {
-			let tabStop = 1;
-			const replaceFunction = (_match: string, p1: string) => '${' + tabStop++ + ':' + p1 + '}';
-			const insertText = p.func.replace(/\[?\$(\w+)\]?/g, replaceFunction);
```

Comment:
This code extrapolated multiple fields for color completions.
But this also limited what was possible.

Color spaces for example have multiple possible keyword values.
Listing these with `|srgb,srgb-linear,...|` is something that helps CSS authors to learn the feature.